src/utils/constants.ts

```diff
@@ -0,0 +1,7 @@
+// Socket events
+export const USER_READY = "userReady";
+export const JOIN_SESSION = "joinSession";
+export const FILE_CLICK = "fileClick";
+export const CURSOR_MOVE = "cursorMove";
+export const DISCONNECT = "userLeave";
+export const SOCKET_URL = "https://8c20-173-33-99-170.ngrok-free.app";
```

Comment:
TODO: this should be replaced by the URL where we host the actual socket

Reply:
https://socket-server-dot-live-sessions-staging.uc.r.appspot.com has been deployed, so I think you can switch this now